lib/src/bloc/movies_bloc.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:async';
+
+import '../model/movies_model.dart';
+import '../resources/movies_repository.dart';
+import 'i_movies_bloc.dart';
+
+class MoviesBloc extends IMoviesBloc {
+  MoviesRepository _moviesRepository = MoviesRepository();
+  StreamController<MoviesModel> _moviesStreamController = StreamController();
+
+  @override
+  void initialize() {}
+
+  @override
+  void dispose() {
+    this._moviesStreamController.close();
```

Comment:
the "this" is not necessary in this call or the other ones below this one, if you remove it it will keep working

Reply:
done